templates/tfengine/components/audit/main.tf

```diff
@@ -23,14 +23,12 @@ terraform {
 }
 
 {{- $parent_field := "org_id"}}
-{{- $parent_var := "var.org_id"}}
+{{- $parent_var := printf "\"%s\"" .parent_id}}
 {{- if eq .parent_type "folder"}}
 {{- $parent_field = "folder"}}
```

Comment:
Can you indent these fields a bit so it's clear it's within the if

Reply:
Done